src/uuid/model/entity/mod.rs

```diff
@@ -404,9 +404,19 @@ impl Entity {
             .map(|children| children.first().cloned())
     }
 
+    /// Returns an array of queried entity ids
+    ///
+    /// Entities with the following properties will not be queried
+    /// - trashed
+    /// - of type applet (type.id 48), article (3), course (7),
+    ///   text-exercise (1), text-exercise-group(4), video(6)
+    ///
+    /// See Issue #144
     async fn find_entity_ids<'a, E>(
         after: Option<i32>,
         instance: Option<&String>,
+        first: Option<i32>,
+        last_modified: Option<&String>,
         executor: E,
     ) -> Result<Vec<i32>, sqlx::Error>
     where
```

Comment:
Idea: We can add the payload struct as an argument as in https://github.com/serlo/serlo.org-database-layer/pull/145/commits/91b3ef8ca17d42a6db5cb23acaa5c5c5dec96673#diff-53630e1968a2dca78c40a46aa0b0cbd39e68220836815c497abf9120f29269b2R97